### DISCUSSION TITLE
Additional models foк Ip adapter

### DISCUSSION THREAD

**Discussion by Garuspik2024** [vladmandic/sdnext#2688] [Q&A] [answered]
Hello. Searched discussion but hasn't found an answer. I'm using ip adapter with SD XL. How I can additional models for this extension?
Haven't found a proper folder.

Thanks in advance and sorry for newbish question

**Comment by vladmandic** [comment]
I haven't seen any additional ip adapters when I implemented this feature, so I haven't enabled manual downloads.

I can add it, but can you send me a link to such an adapter?

**Comment by Garuspik2024** [comment]
Thanks for your answer. Here it is: https://huggingface.co/h94/IP-Adapter/tree/main/sdxl_models
I wanted to try model specific for face ( [ip-adapter-plus-face_sdxl](https://huggingface.co/h94/IP-Adapter/blob/main/sdxl_models/ip-adapter-plus-face_sdxl_vit-h.safetensors)

p.s. And sorry for off topic. In new release of Sd.Next I'll be able to use SD Xl models with original backend? Cause I want to try control.net with XL models

**Reply by vladmandic** [reply, marked as answer]
i'll test those ip adapters first, not every ip adapter can be used in the same manner.

> In new release of Sd.Next I'll be able to use SD Xl models with original backend? Cause I want to try control.net with XL models

no. in the new release, you can use SDXL models with diffusers backend AND use brand new built-in control module (not controlnet extension).

**Comment by Garuspik2024** [comment]
Thanks for your work!

**Reply by vladmandic** [reply]
the problem is that sdxl ip-adapter actually requires separate handling, it cannot be simply loaded using existing code.
i'll add support in the next release (already have a prototype working).
